### ISSUE TITLE
Docs: Avoid usage of long lines in TOML citations because of github issue

### ISSUE THREAD

**Issue by ItsIgnacioPortal** [Politiwatch/privacyspy#158]
**What do you want added to PrivacySpy?**
A clear and concise description of what you're suggesting. (Ex. "I wish _____ would be added to PrivacySpy, because...")
I think we should add a note to [the Contribute page](https://privacyspy.org/contribute/) to tell people to avoid using single-lines citations that are 1024 (or more) characters long, as [the github syntax-highliting does not currently support TOML lines which are that long](https://github.com/github-linguist/linguist/issues/6402).

**Have you considered implementing this addition yourself and submitting a pull request?**
I couldn't find a way to directly contribute to the webpage.

**Comment by doamatto** [comment]
@milesmcc What do you think would be a good way to tackle this ? At a glance, it would make things easier for moderation for those who may not be cloning locally to read the files and instead using GitHub's web editor.

**Comment by milesmcc** [comment]
Yeah, we can absolutely do this. We could enforce it via a new test in our pipeline.
